stetho-okhttp/src/main/java/com/facebook/stetho/okhttp/StethoInterceptor.java

```diff
@@ -46,11 +46,11 @@
 public class StethoInterceptor implements Interceptor {
   private final NetworkEventReporter mEventReporter = NetworkEventReporterImpl.get();
 
-  private final AtomicInteger mNextRequestId = new AtomicInteger(0);
+  private static final AtomicInteger sNextRequestId = new AtomicInteger(0);
```

Comment:
This creates some unique problems of its own since now each interceptor is maintaining its own sequence counting instead of sharing the same singleton state across them all.  You use a static prefix to combat this issue but ultimately what I'd prefer to see is the interceptors centralize and all use a single global which ensures uniqueness.

For instance:

    long nextId = mEventReporter.nextRequestId();

Should solve your problem nicely if all the interceptors (including any custom ones you have) are updated.